kornia/testing/__init__.py

```diff
@@ -119,9 +119,9 @@ def test_exception(self, device, dtype):
     def test_cardinality(self, device, dtype):
         raise NotImplementedError("Implement a stupid routine.")
 
-    @abstractmethod
-    def test_jit(self, device, dtype):
-        raise NotImplementedError("Implement a stupid routine.")
+    # TODO: add @abstractmethod
+    def test_dynamo(self, device, dtype, torch_optimizer):
+        pass
```

Comment:
Maybe have a warning here to encourage the community to update the tests? It will get very verbose

Reply:
I’d avoid that